tests/Unit/Config/XmlConfigFileLoaderTest.php

```diff
@@ -154,7 +160,6 @@ public function test_it_creates_a_valid_config_file_if_all_params_are_present()
         $this->assertInstanceOf(Config::class, $config);
```

Comment:
```suggestion
        self::assertInstanceOf(Config::class, $config);
```

Please fix this in the whole project.

Reply:
Already done, is part of a later pr